### DISCUSSION TITLE
Dark Mode support?

### DISCUSSION THREAD

**Discussion by shivtorov** [EvotecIT/PSWriteHTML#406] [Q&A]
Does PSWriteHTML support dark mode?
https://datatables.net/manual/styling/dark-mode

**Comment by PrzemyslawKlys** [comment]
Adding DarkMode for Datatables is easy. The problem comes from all other elements which also require DarkMode. So first we would need to create a list of elements we support. Then for every element do a check if it supports darkmode or not, and also add some custom darkmode fro panels/sections and so on.

Not easy
